torch/csrc/jit/fuser/cpu/resource_strings.h

```diff
@@ -44,10 +95,24 @@ scalar_t rsqrtf(scalar_t x) {
 
 #define OMP_THRESHOLD 100000
 static void ${kernelName}_kernel(IndexType totalElements, ${formals}) {
-  #pragma omp parallel for if(totalElements > OMP_THRESHOLD)
-  for (IndexType linearIndex = 0;
+  IndexType linearIndex = 0;
+  if (/*vectorizable=*/${vectorizable}) {
```

Comment:
might have to make this a formatting-time distinction if  the vector instructions don't exist.